requirements.txt

```diff
@@ -79,7 +79,7 @@ s3transfer==0.2.1
 setuptools==41.4.0
 six==1.14.0
 sshpubkeys==3.1.0
-task-processing==0.1.17
+task-processing==0.1.18
```

Comment:
Can I convince you to bump to 0.1.19 to grab some additional secret changes that require no tron code changes? :D